chart/values.yaml

```diff
@@ -0,0 +1,64 @@
+image:
+  repository: ghcr.io/cunningpike/fediblockhole
```

Comment:
Why this repo source?

Reply:
It's where I have the built image right now - it should really be moved to your repo and that link updated...